applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/memory_map.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&mram1x {
+	/delete-node/ partitions;
+
+	/* Redefine the "partitions" DTS node. */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Partitions belonging to the MRAM_10 memory block.
+		 * The start of the MRAM_10 memory is reserved for internal IronSide SE firmware.
+		 */
+
+		cpuapp_boot_partition: partition@30000 {
+			reg = <0x30000 DT_SIZE_K(24)>;
+		};
+
+		cpuapp_slot0_partition: partition@36000 {
+			reg = <0x36000 DT_SIZE_K(808)>;
```

Comment:
Consider commenting that the `808` includes size of `cpurad` partition (you could also briefly explain why). I am worried it would be counterintuitive for customers and might not be fixed quickly.
Maybe we could use similar note as in `app_common.dtsi`?

Reply:
Same for `slot1` partitions